### PR TITLE
Created Hydradaboss.json

### PR DESCRIPTION
**Pull request by Hydradaboss**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

**Review by Sindhuinti — APPROVED** [review]
Looks good 🎉

File: data/Hydradaboss.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{
+  "name": "Olaniyi Oluwaseyi",
+  "bio": " Back-End Engineer ",
+   "tags": [
+    "Open Source",
+    "Javascript",
+     "AWS",
+     "Database",
+    "CI/CD",
+    "Freelancing",
```
Comment (sital002) [inline data/Hydradaboss.json:10]:
```suggestion
    "Freelancing",
```
The trailing comma isnot required

File: data/Hydradaboss.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+      "url": "https://github.com/Hydradaboss/recipe-shop",
+      "icon": "FaGithub",
+      "group": "Projects"
+      },
+      {
+      "name": "Chat App",
+      "url": "https://github.com/Hydradaboss/chat_backend",
+      "icon": "FaGithub",
+      "group": "Projects"
+    },
```
Comment (sital002) [inline data/Hydradaboss.json:40]:
```suggestion
    }
```

**Review by sital002 — DISMISSED** [review]
Please make the necessary changes to get your profile merged.

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks for contributing! 🎉

**Pull request merged**